frontend/packages/console-dynamic-plugin-sdk/src/utils/k8s/k8s-resource.ts

```diff
@@ -229,6 +303,15 @@ export const k8sKill = <R extends K8sResourceCommon>(
   );
 };
 
+type OptionsKill<R> = BaseOptions & {
+  model: K8sModel;
+  resource: R;
+  requestInit: RequestInit;
+  json: Record<string, any>;
+};
+
+type K8sKillResource = <R extends K8sResourceCommon>(options: OptionsKill<R>) => Promise<Response>;
```

Comment:
```suggestion
type OptionsDelete<R> = BaseOptions & {
  model: K8sModel;
  resource: R;
  requestInit: RequestInit;
  json: Record<string, any>;
};

type K8sDeleteResource = <R extends K8sResourceCommon>(options: OptionsDelete<R>) => Promise<Response>;
```
We need to pick a side of the fence :)

Reply:
sorry missed, updated :)